operator/src/main/java/oracle/kubernetes/operator/steps/ManagedServersUpStep.java

```diff
@@ -165,7 +163,7 @@ Step createServerStep(
     final Collection<String> servers = new ArrayList<>();
     final Collection<String> preCreateServers = new ArrayList<>();
     final Map<String, Integer> replicas = new HashMap<>();
-    private Step eventStep;
+    private Step failureStep;
```

Comment:
'failureStep' doesn't appear to be assigned; it is always null. Is it needed?

Reply:
I don't think we need it any more. I'll clean it up. Thanks for catching this.

Reply:
done